oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java

```diff
@@ -84,5 +84,29 @@
     private int kafkaHandlerThreadPoolSize;
 
     private int kafkaHandlerThreadPoolQueueSize;
-
+    
+    private String MM2SourceAlias = "";
+
+    private String MM2SourceSeparator = "";
+    
+    public String getTopicNameOfMetrics() {
+        return this.getMM2SourceAlias() + this.getMM2SourceSeparator() + this.topicNameOfMetrics;
+    }
+
+    public String getTopicNameOfProfiling() {
+        return this.getMM2SourceAlias() + this.getMM2SourceSeparator() + this.topicNameOfProfiling;
+    }
+
+    public String getTopicNameOfTracingSegments() {
+        return this.getMM2SourceAlias() + this.getMM2SourceSeparator() + this.topicNameOfTracingSegments;
+    }
+
+    public String getTopicNameOfManagements() {
+        return this.getMM2SourceAlias() + this.getMM2SourceSeparator() + this.topicNameOfManagements;
+    }
+
+    public String getTopicNameOfMeters() {
+        return this.getMM2SourceAlias() + this.getMM2SourceSeparator() + this.topicNameOfMeters;
+    }
```

Comment:
Are you trying to hijack the get method? I recommend you don't do this. Put the logic in the provider, please.

Reply:
OK, working on it.

Reply:
Moved to provider.